tests/utils.go

```diff
@@ -1424,6 +1424,14 @@ func NewRandomVMIWithNS(namespace string) *v1.VirtualMachineInstance {
 
 	t := defaultTestGracePeriod
 	vmi.Spec.TerminationGracePeriodSeconds = &t
+
+	// To avoid mac address issue in the tests change the pod interface binding to masquerade
```

Comment:
there should be an issue reported, linked to in the comment, plus mentioned in PR / commit messages so that the fix is cross-linked accordingly and we can revisit the underlying bug later.

Reply:
done